docs/build.md

```diff
@@ -30,6 +30,65 @@ Selecting this button shows a quick pick list for all the targets CMake Tools is
 
 You can build a single target without changing the current build target from the VS Code command pallette by running the **CMake: Build a target** command, or by pressing the keyboard shortcut (default is `Shift+F7`). CMake will build any dependent targets, even if they aren't directly selected.
 
+## Create a build task
+You can also define a build task from the VS Code command pallette by running the **Tasks: Configure task** command.
+
+![Configure a build task](images/configure_task.png)
+
+By choosing the "CMake: builds" task, the following task will be created in "./.vscode/tasks.json" file.
+
+`{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "cmake",
+			"label": "CMake: build",
+			"command": "build",
+			"targets": [
+				"ALL_BUILD"
+			],
+			"group": "build",
+			"problemMatcher": [],
+			"detail": "CMake template build task"
+		}
+	]
+}`
+
+The default target is set to "all" target (named ALL_BUILD in some generators). You can modify the "targets" value with a single target or a list of targets that can be built. Here is a n example of two customized build tasks in which the name of the task, the list of targets, and the description of the tasks is modified:
+
+`{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "cmake",
+			"label": "Sample CMake build task with single target",
+			"command": "build",
+			"targets": [
+				"prj1"
+			],
+			"group": "build",
+			"problemMatcher": [],
+			"detail": "Build task to build prj1"
+		},
+        {
```

Comment:
The formatting of the example JSON looks incorrect.

Reply:
I used "format selection" on it this time.